src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -147,14 +165,14 @@ private void setupVCFWriter(VCFHeader inputVCFHeader, SampleList samples) {
         vcfWriter = GATKVariantContextUtils.createVCFWriter(outputFile, getReferenceDictionary(), false);
 
         final Set<String> sampleNameSet = samples.asSetOfSamples();
-        final VCFHeader vcfHeader = new VCFHeader(headerLines, sampleNameSet);
+        final VCFHeader vcfHeader = new VCFHeader(headerLines, new TreeSet<>(sampleNameSet));
         vcfWriter.writeHeader(vcfHeader);
     }
 
     @Override
-    public void apply(VariantContext vc, ReadsContext reads, ReferenceContext ref, FeatureContext features ) {
-        ref.setWindow(10,10);
-        final VariantContext mergedVC = merger.merge(Collections.singletonList(vc), vc, includeNonVariants ? ref.getBase() : null, true, false);
+    public void apply(VariantContext variant, ReadsContext reads, ReferenceContext ref, FeatureContext features) {
+        ref.setWindow(10, 10);
```

Comment:
I'm not sure why it's that way... I just copied it from gatk3.  I can check the annotations though.